docs/docs-beta/docs/guides/etl/transform-dbt.md

```diff
@@ -1,4 +1,119 @@
+
+In this file we create:
+- a `DbtProject` object that points to the path of our dbt project
+- a `DbtResource` that references the project object
+- a `@dbt_assets` decorated function that yields Dagster events from the events streamed from the dbt CLI
+- a `Definitions` object that contains our assets and resources
+
+We also use `dbt_project.prepare_if_dev()` to compile our dbt project to ensure Dagster has what it needs to build the asset graph.
+
+## Adding upstream dependencies
+
+Oftentimes, we want Dagster to generate data that will be used by downstream dbt models. Let's add an upstream asset to our `definitions.py` that our dbt project will use as a source:
+
+<CodeExample filePath="guides/etl/transform-dbt/dbt_definitions_with_upstream.py" language="python" title="Adding an upstream asset to definitions.py" />
+
+This asset:
+
+- Pulls customer data from a CSV with pandas
+- Creates a schema called `raw` if it doesn't already exist
+- Writes a `raw_customers` table to the `raw` schema in our DuckDB database
+
+Let's add a dbt model that will source that asset and define the dependency for Dagster, first we'll create our dbt model:
+
+<CodeExample filePath="guides/etl/transform-dbt/jaffle_shop/models/example/customers.sql" language="sql" title="customers.sql" />
+
+Now we'll set up our `_source.yml` file that will point dbt to our upstream asset:
+
+<CodeExample filePath="guides/etl/transform-dbt/jaffle_shop/models/example/_source.yml" language="yaml" title="Adding a _source.yml to our dbt project" />
+
+In this file we need to add the Dagster metadata in the highlighted portion of the code to tell Dagster that this source data is coming from the `raw_customers` asset that we defined earlier. This file now serves two purposes:
+
+1. It's telling dbt where to find the source data for our `customers` model
+2. It's telling Dagster exactly which asset represents this source data
+
+## Adding downstream dependencies
+
+Similarly, we often have assets that depend on the output of our dbt models. Let's create an asset that depends on the result of our new `customers` model, this asset will create a histogram of the first names of the customers:
+
+<CodeExample filePath="guides/etl/transform-dbt/dbt_definitions_with_downstream.py" language="python" title="Adding an downstream asset to definitions.py" />
+
+The important line to note is where we set the dependency of our asset to our customers model using the `get_asset_key_for_model` function:
+
+```python
+deps=get_asset_key_for_model([dbt_models], "customers")
+```
+
+This line finds our `customers` dbt model, gets its corresponding asset key, and directly sets it as a dependency of our new `customer_histogram` asset.
+
+## Scheduling dbt models
+
+We can easily schedule our dbt models using the Dagster dbt integration's `build_schedule_from_dbt_selection` function:
```

Comment:
```suggestion
You can schedule your dbt models by using the Dagster dbt integration's `build_schedule_from_dbt_selection` function:
```

Also, try to avoid words like `easily`, `simply`, etc. If a user is having trouble with it and comes to the docs frustrated, that type of language can make them feel worse about it